src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -51,6 +57,15 @@
 
 public class GtfsModule implements GraphBuilderModule {
 
+  public static final Set<Class<?>> faresV2Entities = Set.of(
```

Comment:
Naming convention for constants, use upper case.

Reply:
Done.